sdk/search/azure-search-documents/azure/search/documents/_headers_mixin.py

```diff
@@ -10,11 +10,11 @@
 class HeadersMixin:
     @property
     def _headers(self) -> Dict[str, Any]:
+        if self._aad:  # type: ignore
+            return {"Accept": self._ODATA_ACCEPT}  # type: ignore
```

Comment:
Was the lack of this header in the AAD case a bug that is fixed? (Is this changelog-worthy?)